### ISSUE TITLE
Include climate temperature in area average temperature sensor and adopt unit_of_measurement

### ISSUE THREAD

**Issue by klatka** [jseidl/hass-magic_areas#61]
Thank you for this component. Actually, I want to integrate it into my config.

I have an area with only a climate sensor. Although this climate sensor has a temperature sensor, no `Area: Temperature (AreaName)` is created through magic_areas.

If I add this temperature sensor manually in `include_entities` a new area temperature sensor appears:
`sensor.area_temperature_schlafzimmer`

The problem is that this sensor has no `unit_of_measurement`:
```
sensors:
  - sensor.schlafzimmer_thermostat_temperature
friendly_name: Area Temperature (Schlafzimmer)
device_class: temperature
```
Other area temperature sensors with sensors that are auto included the unit is adopted.
The source sensor `sensor.schlafzimmer_thermostat_temperature` has a valid `unit_of_measurement`.

1. Please automatically include all sensors of a climate device.
2. Please adopt unit_of_measurement for manually included devices.

Edit: Same with humidity sensor of the climate entity. `unit_of_measurement` is also missing at interior sensor.

**Comment by jseidl** [comment]
This is a bug. `unit_of_measurement` should always be copied through. Will investigate.

Currently only `sensor` and `binary_sensor` devices are supported. Adding `climate` temperature extraction can be a future feature.

**Comment by jseidl** [comment]
Possibly fixed by #74, released on [2.0.0](https://github.com/jseidl/hass-magic_areas/releases/tag/v2.0.0).

Can you test it out and let me know if the issue persists?

Make sure you add `unit_of_measurement` via `customize`. I'm still not support climate devices as temperature sensors yet.

**Comment by klatka** [comment]
`unit_of_measurement` is being adopted correctly in `2.0.x`
